dataflux_core/tests/fake_gcs.py

```diff
@@ -69,6 +72,13 @@ def _add_file(self,
                                     self,
                                     storage_class=storage_class)
 
+    def test_iam_permissions(self, permission: any):
```

Comment:
this should probably be called "check_iam_permissions" unless this is in itself a unit test. Additionally this code can be condensed into `return [p for p in permissions if p in self.permissions]`

Reply:
test_iam_permissions is an inbuilt function. 
Updated the function. Thanks Jd!